### DISCUSSION TITLE
gnuradio: update to 3.10.x

### DISCUSSION THREAD

**Discussion by citizenrich** [Homebrew/discussions#3059] [Tap maintenance and Homebrew development]
Hi folks. I'd like to ask for help to update the GNU Radio formula to the latest release series which is 3.10. https://github.com/gnuradio/gnuradio/releases The current formula is pinned to 3.9.3.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew
